subsys/nrf_security/src/drivers/cracen/cracenpsa/include/cracen_psa.h

```diff
@@ -361,4 +361,13 @@ psa_status_t cracen_spake2p_get_shared_key(cracen_spake2p_operation_t *operation
 
 psa_status_t cracen_spake2p_abort(cracen_spake2p_operation_t *operation);
 
+int ed25519_sign(const uint8_t *ed25519, char *signature,
```

Comment:
It is strange to use int for the returns on these specific ones, instead of psa_status_t, but these returns are silex error codes and not psa ones so using psa_status_t would be wrong. Using int as that is what sicrypto used, but am open to better ideas.